index.js

```diff
@@ -0,0 +1,88 @@
+/* Set Date to Current Date */
+Date.prototype.toDateInputValue = (function() {
+  var local = new Date(this);
+  local.setMinutes(this.getMinutes() - this.getTimezoneOffset());
+  return local.toJSON().slice(0,10);
+});
+
+document.getElementById('implementationDate').value = new Date().toDateInputValue();
+
+/* Contact Form Validation */
```

Comment:
I tend to not comment the code if it's obvious from the code as to what it's doing. Variable/function names should be able to explain most of it.

Reply:
Also, we tend to use `//` for JavaScript comments instead of `/*` because it's easier to uncomment certain bits on multi line comments.